internal/common/vote/vote.go

```diff
@@ -0,0 +1,191 @@
+package vote
+
+import (
+	"math/rand"
+	"time"
+)
+
+func GetVotesForRule(ruleID string, islandID int) map[int][]int {
+	votes_layout := make(map[int][]int)
```

Comment:
I believe we use camelCase as our naming convention for locals as per go standards, best to keep it consistent. This should be resolved throughout the PR